apps/src/userHeaderEventLogger/userHeaderEventLogger.js

```diff
@@ -0,0 +1,5 @@
+import $ from 'jquery';
+
+$(document).ready(function () {
+  console.log('User header event logger');
```

Comment:
Wait, won't this log to the console on every page? (that's not a big deal tbh, but curious because the description says it doesn't?)

Reply:
Oh it won't log on any page because the haml view `user_header_event_logger` isn't in the user header (that will be added in the next PR). Currently, this is just a pair of `haml`/`js` files that aren't being used anywhere yet.

Reply:
lol right 🤦‍♀️